config/locales/en.yml

```diff
@@ -883,6 +883,7 @@ en:
         new_feed_url: If your podcast feed is moving, use this field to point to the new URL where your podcast is located. The current feed should be maintained until all of your subscribers have migrated.
         private: Your feed can be public or private. Leave unchecked to keep it a Public Feed. If you check this box this feed becomes Private and you'll need to generate an authorization token. Be careful who you share your private URL with - <strong>anyone who has access to the URL will be able to use it</strong>.
         url: If you already have a public URL for your podcast feed (e.g., <a href="https://support.google.com/feedburner/answer/78475?hl=en" target="_blank">feedburner</a>), enter it here. It should point to your <a href="%{published_url}" target="_blank">private feed URL</a>.
+        exclude_episodes: Exclude all episodes from the default feed from being automatically added to this feed.
```

Comment:
@radical-ube I'm finding the label "Exclude Default Episodes" a little confusing. The help text is clear. So maybe we can change the label to:

"Exclude Default Feed Episodes" or "Skip Default Feed Episodes"